hassio/hassio-main.html

```diff
@@ -43,59 +36,34 @@
         show-menu='[[showMenu]]'
       ></hass-loading-screen>
     </template>
+
     <template is='dom-if' if='[[loaded]]'>
-      <iron-pages
-        selected='[[_routeData.page]]'
-        attr-for-selected='page-name'
-        fallback-selection='not-found'
-        selected-attribute='visible'
-      >
-        <hassio-addon-view
-          page-name='addon'
+      <app-header-layout has-scrolling-region>
+        <app-header fixed slot='header'>
+          <app-toolbar>            
+            <ha-menu-button narrow='[[narrow]]' show-menu='[[showMenu]]'></ha-menu-button>
+            <paper-icon-button
+              icon='mdi:arrow-left'
+              on-tap='backTapped'
+              hidden='[[!pageIsAddon]]'
+            ></paper-icon-button>
+            <div main-title>[[title]]</div>
+          </app-toolbar>
+          <template is='dom-if' if='[[!pageIsAddon]]'>
```

Comment:
This is still a very weird coupling.

I would expect it to work like this inside `hassio-main`:

```
if showing addon:
  <hassio-addon>
end

if not showing addon:
  <hassio-pages-with-tabs>
end
```

Within `hassio-pages-with-tabs`:

```
if dashboard:
  <hassio-dashboard>
end

if bla:
  <hassio-bla>
end
```